phy/cluster/manual/session.py

```diff
@@ -305,6 +287,43 @@ def __init__(self, store_path=None):
 
         self.connect(self.on_open)
         self.connect(self.on_cluster)
+        self.connect(self.on_close)
+
+    # Settings
+    # -------------------------------------------------------------------------
+
+    def get_user_settings(self, key):
+        return self.settings_manager.get_user_settings(key,
+                                                       scope='experiment')
+
+    def set_user_settings(self, key=None, value=None,
+                          path=None, file_namespace=None):
+        return self.settings_manager.set_user_settings(
+            key, value, scope='experiment', path=path,
+            file_namespace=file_namespace)
+
+    def get_internal_settings(self, key):
+        return self.settings_manager.get_internal_settings(key,
+                                                           scope='experiment',
+                                                           )
+
+    def set_internal_settings(self, key, value):
+        return self.settings_manager.set_internal_settings(key,
+                                                           value,
+                                                           scope='experiment',
+                                                           )
+
+    def _load_default_settings(self):
+        """Load default settings for manual clustering."""
+        curdir = op.dirname(op.realpath(__file__))
+        # This is a namespace available in the config file.
+        file_namespace = {
+            'n_spikes': self.model.n_spikes,
```

Comment:
we can add more later: the point is that you can have access to the `n_spikes` variable within the settings file